src/etc/rust_analyzer_settings.json

```diff
@@ -16,7 +16,7 @@
         "compiler/rustc_codegen_gcc/Cargo.toml"
     ],
     "rust-analyzer.rustfmt.overrideCommand": [
-        "./build/host/rustfmt/bin/rustfmt",
+        "${workspaceFolder}/build/host/rustfmt/bin/rustfmt",
         "--edition=2021"
     ],
     "rust-analyzer.procMacro.server": "./build/host/stage0/libexec/rust-analyzer-proc-macro-srv",
```

Comment:
@Veykril  will that variable also work for this setting? What about `check.overrideCommand` and `buildScripts.overrideCommand`? We should probably use it everywhere?

Reply:
Yes it should work in all VSCode settings.